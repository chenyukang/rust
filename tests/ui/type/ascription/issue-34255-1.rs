struct Reactor {
    input_cells: Vec<usize>,
}

impl Reactor {
    pub fn new() -> Self {
        input_cells: Vec::new()
        //~^ ERROR failed to resolve: use of undeclared crate or module
    }
}

// This case isn't currently being handled gracefully, including for completeness.
fn main() {}
