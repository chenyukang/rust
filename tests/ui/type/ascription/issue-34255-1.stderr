error[E0433]: failed to resolve: use of undeclared crate or module `input_cells`
  --> $DIR/issue-34255-1.rs:7:9
   |
LL |         input_cells: Vec::new()
   |         ^^^^^^^^^^^ use of undeclared crate or module `input_cells`
   |
help: consider importing this struct
   |
LL | use std::vec::Vec;
   |
help: if you import `Vec`, refer to it directly
   |
LL -         input_cells: Vec::new()
LL +         Vec::new()
   |

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
